src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowEngine.java

```diff
@@ -222,10 +216,18 @@ public static <T> List<DcLoadFlowResult> run(T network, LfNetworkLoader<T> netwo
     }
 
     /**
-     * A simplified version of DcLoadFlowEngine that supports on the fly bus and branch disabling and that do not
-     * update the state vector and the network at the end (because we don't need it to just evaluate a few equations).
+     * A simplified version of DcLoadFlowEngine that supports on the fly bus and branch disabling.
+     * Note that it does not update the state vector and the network at the end (because we don't need it to just evaluate a few equations).
      */
     public static double[] run(DcLoadFlowContext loadFlowContext, DisabledNetwork disabledNetwork, ReportNode reportNode) {
+        return run(loadFlowContext, disabledNetwork, reportNode, new ArrayList<>());
+    }
```

Comment:
Better to use Collections.emptyList() here

Reply:
Indeed. Thank you.